README.md

```diff
@@ -42,7 +42,69 @@ PM > pip3 install --upgrade --pre arcus-azureml
 ```
 
 # Local development
-    
+
+The following steps show you how you can get started with local development for the Arcus Python packages.  These steps have been tested and verified on a Windows 10 development environment, but the same steps should be applied in case of developing on MacOS or Linux.
+
+## Installation guide
+
+The steps below are based on the [official documentation of Microsoft](https://docs.microsoft.com/en-us/windows/python/web-frameworks).
+
+### Install Windows Subsystem for Linux
+
+Follow the steps, outlined in [this document](https://docs.microsoft.com/en-us/windows/wsl/install).
+
+- Enable the Windows Feature for WSL (if not done yet)
+- Run `wsl --install -d Ubuntu` in the command prompt
+- Configure your user and password for your Linux Distribution and update that distribution, using `sudo apt update && sudo apt upgrade`
+- Install Windows Terminal
+- Following any (optional) extra instructions in the WSL development [environment documentation](https://docs.microsoft.com/en-us/windows/wsl/setup/environment#set-up-your-linux-username-and-password).
+
+### Install and verify required Python prerequisites
+
+- Verify python is installed in your Linux distribution by running `python3 --version`.  This guide has been tested with **Python 3.8.10**.
+- Install **pip**, which is the Python Package installer.  Execute `sudo apt install python3-pip`.
+- Install **venv**, which supports to create Virtual Python Environments (and isolate dependencies from each other).  Execute `sudo apt install python3-venv`.
+- When running `deactivate`, you deactive the virtual environment.
+
+### Create your Arcus python environment
+
+- Get the correct code from Github, by performing a `git clone` into your WSL.
+- Install all required dependencies for your arcus project, by executing the following in your WSL prompt: `pip3 install -r requirements.txt`.  This takes all packages that are configured in the requirements.txt file and installs them in your virtual environment.
+- Execute the following in the WSL shell: `python3 -m venv .venv`.  This creates a new virtual Python environment, in a directory named **.venv**.  This can be done for every Python project you develop.  
+- Activate and start your .arcus environment by running `source .venv/bin/activate`. 
+- Exclude `.venv` from your `.gitignore`, if not done yet. 
+
+### Configure Visual Studio Code
+
+In order to get started developing locally, we recommend [Visual Studio Code](https://code.visualstudio.com/) as the main tool for development.  If you are new to this, we recommend reading this [overview and getting started guide](https://code.visualstudio.com/docs/python/python-tutorial).
```

Comment:
We shouldn't actually force people to use any IDE, IMHO, so maybe something we should consider adapting so we can support any.

Reply:
Agree it should not be forced.  That step could be made optional.  
But considering that many people are using VSCode, it could be a good option to at least describe 'one way of working', and mention any editor that supports Python will work.
Will update accordingly